packages/design-system/src/styles/components/_HelpDrawer.scss

```diff
@@ -54,6 +62,21 @@
   }
 }
 
+.ds-c-help-drawer__header-container {
+  align-items: flex-start;
+  background-color: $help-drawer-header-fill;
+  display: flex;
+  padding: $help-drawer-padding;
+}
+
+.ds-c-help-drawer__header-heading {
```

Comment:
The utility classes you replaced with this includes a `ds-u-text--lead`. Does that need to be added here?

Reply:
Turns out `ds-u-text--lead` wasn't a real class. I think the class that was wanted was `ds-text--lead`, but since the style wasn't applied, I assumed this class was unnecessary. 

On that point, it was a little confusing seeing some text helper classes that appeared to be utilities not have that `-u` identifier. I encountered similar confusion font-sizing classes for various media breakpoints. It's probably 100% my lack of familiarity with our util classes, but it wasn't immediately intuitive what the class was accomplishing. Is this a problem? I dunno. Just wanted to point it out.

Reply:
That's a good point. It totally seems like a utility class, but they appear to be in a category of their own and treated as first-class citizens. @line47, what do you think about this? Does it make sense to you for these typography classes to omit the `-u` prefix? The other thing I can think to compare it to are the layout classes with `-l`. If we did want typography to be separate, shouldn't we continue the pattern with something like `-t` for typography? All that said, I'm not eager to make breaking changes 😆